app/src/main/java/com/appdev/eateryblueandroid/ui/components/home/TopBar.kt

```diff
@@ -0,0 +1,33 @@
+package com.appdev.eateryblueandroid.ui.components.home
```

Comment:
Should this class be deleted? I don't think it is doing much

Reply:
oops forgot to push it haha